internal/privatemessaging/operations.go

```diff
@@ -129,6 +129,7 @@ func (pm *privateMessaging) PrepareOperation(ctx context.Context, op *core.Opera
 		if err != nil {
 			return nil, err
 		}
+		batch.Namespace = pm.namespace.NetworkName
```

Comment:
Great find.

So when we put the network namespace concept in here:
[dispatchBatchCommon](https://github.com/kaleido-io/firefly/blob/main/internal/privatemessaging/privatemessaging.go#L184)

We missed it in the recovery of a prepared operation path.